nullaway/src/main/java/com/uber/nullaway/CodeAnnotationInfo.java

```diff
@@ -139,30 +157,39 @@ public boolean isClassNullAnnotated(Symbol.ClassSymbol classSymbol, Config confi
    *     annotations on enclosing classes, the containing package, and other NullAway configuration
    *     like annotated packages
    */
-  private CacheRecord get(Symbol.ClassSymbol classSymbol, Config config) {
-    CacheRecord record = cache.getIfPresent(classSymbol);
+  private ClassCacheRecord get(Symbol.ClassSymbol classSymbol, Config config) {
+    ClassCacheRecord record = classCache.getIfPresent(classSymbol);
     if (record != null) {
       return record;
     }
     if (classSymbol.getNestingKind().isNested()) {
+      Symbol owner = classSymbol.owner;
+      Preconditions.checkNotNull(owner, "Symbol.owner should only be null for modules!");
+      Symbol.MethodSymbol enclosingMethod = null;
+      if (owner.getKind().equals(ElementKind.METHOD)
+          || owner.getKind().equals(ElementKind.CONSTRUCTOR)) {
+        enclosingMethod = (Symbol.MethodSymbol) owner; // Potentially null!
```

Comment:
What does the `// Potentially null!` comment mean?  Doesn't `owner` get dereferenced right in the `if` condition above this?

Reply:
Actually, this comment is wrong, left-over from a different implementation. Removing.